README.md

```diff
@@ -322,6 +323,20 @@ Now that you've got a local Conjur instance running, what can you do with it?
 Try some of our [tutorials](https://www.conjur.org/get-started/tutorials/) on
 Conjur.org.
 
+### Configuring Conjur with predefined admin password
+
+The following command will allow you to specify the admin user's password:
+```
+docker-compose exec conjur bash -c 'echo -n "MySecretP@SS1" | conjurctl account create --password-from-stdin --name  myConjurAccount'
+```
+The password must be provided via STDIN in any manner you prefer and must meet
+the following complexity rules:
+- Between 12 and 128 characters
+- 2 uppercase letters
+- 2 lowercase letters
+- 1 special character
+- 1 digit
+
```

Comment:
We should probably note that this only works for Conjur OSS versions greater than 1.11.4

Next week I'll release Conjur, perhaps we can hold this off until then? At that point we can authoritatively say "Conjur v1.12.0+" or something

Reply:
This can be held off till after the release.